src/styles/main.scss

```diff
@@ -0,0 +1,39 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 200px;
+}
+
+.watch {
+  position: relative;
+
+  &__line-second {
+    position: absolute;
+    top: 0;
+    left: 50px;
+    width: 4px;
+    height: 200px;
+    background-color: aquamarine;
+    transform-origin: bottom left;
+    animation: sec 1s infinite;
+  }
+
+  &__line-minute {
+    position: absolute;
+    top: 100px;
+    left: 50px;
+    width: 6px;
+    height: 100px;
+    background-color: fuchsia;
+    transform-origin: bottom left;
+    animation: sec 60s infinite;
+  }
```

Comment:
your watch hands have a lot of similar styles. Create either common class with common css rules or use scss mixin